### PR TITLE
Expose ports in Docker instructions

### PR DESCRIPTION
**Pull request by tbg**
[Review on Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/4334)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ Other operating systems will have a similar set of commands. Please check Docker
 Pull the CockroachDB Docker image and drop into a shell within it:
 ```bash
 docker pull cockroachdb/cockroach
-docker run -t -i cockroachdb/cockroach shell
+docker run -p 26257:26257 -p 15432:15432 -t -i cockroachdb/cockroach shell
```
Comment (bdarnell) [inline README.md]: I think if we put `EXPOSE 26257 15432` in our `Dockerfile` then we could simplify this to `docker run -P`.

**Comment by bdarnell** [comment]
LGTM

File: README.md [hunk 2]
```diff
@@ -69,6 +69,12 @@ Setting up Cockroach is easy, but starting a test node is even easier. All it ta
 ./cockroach start --dev &
 ```
 
+Verify that you're up and running by visiting the cluster UI. If you're running
+without Docker (or on Linux), you'll find it at
+[localhost:26257](http://localhost:26257); for OSX, things are a little more
```
Comment (tamird) [inline README.md]: this wording suggests that you always need to go to `docker-machine ip default` on OSX (regardless of docker). of course that would be nonsense, but y'know.

File: README.md (hunk 1 above)
Comment (tschottdorf) [inline README.md]: No, `-P` exposes on random ports. No bueno.

File: README.md (hunk 2 above)
Comment (tschottdorf) [inline README.md]: changed to `on OSX and Docker`.

**Pull request merged**